pages/FilePage.tsx

```diff
@@ -187,13 +187,13 @@ export function FilePage() {
                   }
                 }
               })
-              .then((objson) => {
+              .then((objson: React.SetStateAction<{ result: string; }>) => {
```

Comment:
Do you think `objson` name makes sense @filiptronicek 🤔

Reply:
Instead, you can use `result` or `response`

Reply:
Fair enough, good point